### ISSUE TITLE
accessibility impacts should be considered in principles

### ISSUE THREAD

**Issue by npdoty** [patcg/docs-and-reports#63]
One issue may be mitigations against deceptive patterns in advertising, or for consent determination.

We also may just want to work through this checklist: https://w3c.github.io/apa/fast/checklist.html

ask for review from APA WG: https://www.w3.org/WAI/about/groups/apawg/

**Comment by wareid** [comment]
This resource might be helpful as you explore this: https://www.deceptive.design/

**Comment by martinthomson** [comment]
I found this taxonomy study pretty helpful.  Or rather, the output of it in terms of the high level groupings of practices.  Note the use of terminology is still contested in this area.

https://www.researchgate.net/publication/374165695_An_Ontology_of_Dark_Patterns_Foundations_Definitions_and_a_Structure_for_Transdisciplinary_Action

Funny, after loading this page, the site asked rather aggressively if I were interested in research into **Darkness**, which probably classifies under Forced Action somewhere.
